spt/OrangeBox/modules/ClientDLL.cpp

```diff
@@ -806,26 +837,16 @@ void __fastcall ClientDLL::HOOKED_AdjustAngles_Func(void* thisptr, int edx, floa
 		}
 
 		Friction(pl, onground, vars);
-		Strafe(pl, vars, onground, jumped, ((*reinterpret_cast<int*>(reinterpret_cast<uintptr_t>(player) + offFlags)) & FL_DUCKING), type, dir, tas_strafe_yaw.GetFloat(), va[YAW], out, reduceWishspeed, btns, usingButtons);
 
-		//EngineDevMsg("[Strafing] Yaw = %.8f\n", out.Yaw);
+		if (tas_strafe_vectorial.GetInt() == 1)
```

Comment:
It's better to use `GetBool()` here since we only care about the cvar being "enabled" or "disabled".